getting-started.md

```diff
@@ -7,218 +7,224 @@ description: This tutorial will help you run UniRep examples
-    -p Unirep.epk.publicSignals.WyIxMDU0Njc5NTYwMzE4NjkzMzI2MTU5NTQ5NjA3ODAwMDE3MTc0NDQ4OTE0NDIwMDg2MzY0ODU4NTM4ODQxMzIzNTg5MDc5NzcyNDIyOSIsIjEiLCIxMjU1ODYzMTQxIl0
+    -x $UNIREP_CONTRACT_ADDRESS  \
+    -pf $EPOCH_KEY_PROOF  \
+    -p $EPOCH_PUBLIC_SIGNALS
 ```
+- NOTE: `-p` is the public signals of the epoch key proof, and `-pf` is the epoch key proof
 
-* NOTE: `-p` is the public signals of the epoch key proof, and `-pf` is the epoch key proof
 
 ### 8. Submit epoch key proof to Unirep smart contract
-
-```
+```bash
 npx ts-node cli/index.ts submitEpochKeyProof \
-    -x 0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6  \
-    -d 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80  \
-    -pf Unirep.epk.proof.WyI1Mjc1ODYyNTY3NzYwNDM4ODI5NjE2OTcxMzgyNTE3MDIzODgzMDc3MDI3MzExOTE5ODQwMTQyNDQyODg4ODM4OTcwNjc0MTcyMzMyIiwiOTc2NTA4NTY3MTY4MjkzOTMzODc0MzQ0OTM5MjAyNjY1NjcxOTUyNzE1MjA2NDc5ODc3MzMxNDAyNjM0NzUxODM1NTExNjk2MTM4OSIsIjE3NzU1NzE2ODYxMjgzMzM5MzA4NzY0NDA2NDA1MjMxNjcwNjAzNDUzMTA1NTQzNzE3NjE5MjEwOTM1MTg2MTE5MDkwOTc5OTU3OTcxIiwiNzIzMTA0NDkwMzk5MzQ0MzM0NDM0MTY1NTkyOTAzNjE2MjEzNTgwOTQxNzQ2NDA1NTcyMjc5MjYzODQyNTc2NDE1MDM1MzE4MzQwOCIsIjc3Mjk3NzExNjMwNTEzOTYxNzczMjg1NjczMTc3NDA3MTAwMTI0NzEwOTI1NTAxNzIyMzU0OTc3NjgyMzI5MzUwNjExNzUzODgyIiwiMTExOTk3MjI0NDM0MzMxMDkwNTY0NjYzMTQzMzQ3MTYxNjMzODMyNTcyOTIzMjkwMDA1NTg3ODQxMTE0OTA3NDAzMTEzMTMyNzM5NzMiLCIyMTU4OTE5NTA4NjYzODY4MzY2MDMwNTAzOTgzNTk1MTA5NzMwMDE2MDA0MjI5NzUxODgwOTE2OTA3MzMwNjQ2ODMyNjM5NTczNTEyMyIsIjMxNTA1OTE3NjczNDI4NzcxMTI2OTQxMDY0NjYxNTg3NDM4NzcyOTkyMzk1MzY5ODc4MjcxMjkwMzQ2MTU5MjgxMTIyOTAwMTE2NTYiXQ  \
-    -p Unirep.epk.publicSignals.WyIxMDU0Njc5NTYwMzE4NjkzMzI2MTU5NTQ5NjA3ODAwMDE3MTc0NDQ4OTE0NDIwMDg2MzY0ODU4NTM4ODQxMzIzNTg5MDc5NzcyNDIyOSIsIjEiLCIxMjU1ODYzMTQxIl0
+    -x $UNIREP_CONTRACT_ADDRESS  \
+    -d $USER_PRIVATE_KEY  \
+    -pf $EPOCH_KEY_PROOF  \
+    -p $EPOCH_PUBLIC_SIGNALS
 ```
-
-* NOTE: `-d` is the user's private key here
-* The proof index will be printed, for example:
-
+- NOTE: `-d` is the user's private key here
+- The proof index will be printed, for example:
 ```
 Proof index:  2
 ```
-
-* Then the epoch key with the proof index should be handed to attester to be attested.
+- Then the epoch key with the proof index should be handed to attester to be attested.
 
 ### 9. Attester attest to epoch key
-
-```
+```bash
 npx ts-node cli/index.ts attest \
-    -x 0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6  \
-    -d 0x59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d  \
-    -epk 1255863141  \
+    -x $UNIREP_CONTRACT_ADDRESS  \
+    -d $ATTESTER_PRIVATE_KEY  \
+    -epk $EPOCH_KEY  \
```

Comment:
I think the `EPOCH_KEY` only used once so there is no need to export

Reply:
However, user can generate another epoch key and can reuse this command.